lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js

```diff
@@ -29,7 +30,8 @@ import { QC_SUMMARY_COLORS } from '../../../components/qcFlags/qcSummaryColors.j
  */
 const runDetectorAsyncQualityDisplay = (runNumber, dplDetectorId, { dataPassId, simulationPassId }, qcSummary) => {
     if (!qcSummary?.[runNumber]?.[dplDetectorId]) { // If there is no QC flag assigned
-        return qcFlagCreationPanelLink({ dataPassId, simulationPassId, runNumber, dplDetectorId });
+        return qcFlagCreationPanelLink({ dataPassId, simulationPassId, runNumber, dplDetectorId })
```

Comment:
First, I think that such thing should go to to the `qcFlagCreationPanelLink`
Then, here you return the tooltip even in the case where dpl detector list is loading or has an error... (You don't use other in this case, but this can give the intuition why remote data cases should be handled carefully)

Reply:
refactored